py/node/Factory.py

```diff
@@ -21,6 +21,19 @@ def create(data = 0):
         elif (seed == SUBTRACT):
             return SubtractorNode(data)
 
+    @staticmethod
+    def createType(seed = 0, data = 0):
+        if (seed == ORDER):
+            return OrderNode(data)
+        elif (seed == DIVIDE):
+            return DivisorNode(data)
+        elif (seed == MULTIPLY):
+            return MultiplierNode(data)
+        elif (seed == SUM):
+            return SummationNode(data)
+        elif (seed == SUBTRACT):
+            return SubtractorNode(data)
+
```

Comment:
If you end up removing the commented out code I commented on above, then get rid of this from this PR aswell (as it looks like the only bit of code that is using it)